templates/user/Profile.html

```diff
@@ -0,0 +1,137 @@
+{% extends "user/AccountSettings.html" %}
+
+{% block title %}{{ user.full_name() }} - Profile{% endblock %}
+
+{% block settings_content %}
+        <div class="s12">
+            <div class="card">
+                <div class="card-content">
+                    <div class="row">
+                            {% if user.photo is none %}
+                                <div class="col s4">
+                                    <img class="circle responsive-img profile-img z-depth-1"
+                                         src="https://goo.gl/WrdnWG" alt="Face">
+                                </div>
+                            {% else %}
+                                <div class="col s4">
+                                    <img class="circle responsive-img profile-img z-depth-1"
+                                         src="{{ url_for('static', filename='photos/')}}{{ user.photo }}"
+                                         alt="Face">
+                                </div>
+                            {% endif %}
+                        <div class="col s6 offset-s2">
+                            <span class="card-title grey-text text-darken-4 activator">
+                                {{ user.first_name }} {{ user.last_name }}<i class="material-icons right">more_vert</i>
+                            </span>
+                            <p class="medium-small grey-text">
+                                {% if user.specialist is not none %}
+                                    {% for s in user.specialist.services.all() %}
+                                        {{ s.title }} <br>
+                                    {% endfor %}
+                                {% endif %}
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-reveal">
+                    <p class="flow-text">
+                    <h4 class="card-title grey-text text-darken-4">
+                        {{ user.first_name }} {{ user.last_name }}<i class="mdi-navigation-close right"></i>
+                    </h4>
+                    <span>
+                        {% if user.specialist is not none %}
```

Comment:
Basically I think you can write `{% if user.specialist %}`. It's shorter and still pretty explicit